lib/index.js

```diff
@@ -12,6 +12,13 @@
 var shallowCopy = require('./utils').shallowCopy;
 
 
+/**
+ * creates a HttpClient or a flavor of it.
```

Comment:
A flavor? What kind of flavour? Watermelon? Mint? All kidding aside, might be worth elaborating on what that is.